pennylane_lightning/core/src/simulators/lightning_gpu/utils/cuda_helpers.hpp

```diff
@@ -14,6 +11,9 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+// Adapted from JET: https://github.com/XanaduAI/jet.git
+// and from Lightning: https://github.com/PennylaneAI/pennylane-lightning.git
```

Comment:
We can remove this as it isn't needed anymore

Reply:
Done!